projectroles/tests/test_views.py

```diff
@@ -3123,6 +3123,81 @@ def test_render_not_found(self):
             )
         self.assertEqual(response.status_code, 404)
 
+    @override_settings(
+        PROJECTROLES_ALLOW_LOCAL_USERS=False,
+        ENABLE_SAML=False,
+        AUTH_LDAP_USERNAME_DOMAIN='',
```

Comment:
Note that this settings value is never set or read unless `ENABLE_LDAP` is true.